pep-0484.txt

```diff
@@ -276,6 +276,34 @@ implemented by deferring to ``isinstance(x, collections.abc.Callable)``.
 However, ``isinstance(x, typing.Callable[...])`` is not supported.
 
 
+Decorators
+----------
+
+Decorators can modify the types of the functions or classes they
+decorate. Use the `decorated_type` decorator to declare the type of
```

Comment:
I think it is better to use double backquotes:
``` 
``decorated_type``
```
here and in few places below (this will make the reST style consistent with the rest of the PEP).